lib/ASTGen/Sources/ASTGen/Exprs.swift

```diff
@@ -1,17 +1,22 @@
 import CASTBridging
-import SwiftParser
 import SwiftSyntax
 
 extension ASTGenVisitor {
   public func visit(_ node: ClosureExprSyntax) -> ASTNode {
-    let statements = node.statements.map { self.visit($0).bridged }
-    let body: UnsafeMutableRawPointer = statements.withBridgedArrayRef { ref in
-      let startLoc = bridgedSourceLoc(for: node.leftBrace)
-      let endLoc = bridgedSourceLoc(for: node.rightBrace)
-      return BraceStmt_create(ctx, startLoc, ref, endLoc)
-    }
-
-    return .expr(ClosureExpr_create(ctx, body, declContext))
+    .expr(
+      node.statements.map {
+        self.visit($0).bridged
+      } withBridgedArrayRef: { statements in
+        let body = BraceStmt_create(
+          self.ctx,
+          self.bridgedSourceLoc(for: node.leftBrace),
+          statements,
+          self.bridgedSourceLoc(for: node.rightBrace)
+        )
+
+        return ClosureExpr_create(self.ctx, body, self.declContext)
```

Comment:
Can you put a TODO or FIXME in here to translate the signature of a closure? (parameter list, captures, etc.)